src/styles.css

```diff
@@ -9,4 +9,12 @@ body {
   justify-content: center;
   align-items: center;
   border-bottom: 1px solid black;
+}
+
+.footer {
+  height: 50px;
```

Comment:
The total height of the footer is currently 51px and not 50px. That's because it has a border of 1px is added to the element's height